salt/metalk8s/orchestrate/downgrade/init.sls

```diff
@@ -133,6 +133,23 @@ Sync module on salt-master:
     - name: saltutil.sync_all
     - saltenv: metalk8s-{{ dest_version }}
 
+{#- In previous MetalK8s releases, Cluster and Service configurations where not
+    persisted throughout. Now in `2.5.0` we have added ConfigMaps to store
+    service configurations. When running a downgrade to a version inferior to
+    `2.5.0` we do not need to deploy these ConfigMap objects. #}
+{#- Only run this step if `dest_version < 2.5.0` #}
```

Comment:
No, we want to run it only if `dest_version >= 2.5.0` :)

Reply:
Thinking this still works because we would have already deployed the ConfigMaps in the Bootstrap orchestrate before the downgrade.